mtp_cashbook/apps/cashbook/tests/test_functional.py

```diff
@@ -20,6 +21,8 @@ def _databases_names(cls, include_mirrors=True):
     def setUp(self):
         path = './node_modules/phantomjs/lib/phantom/bin/phantomjs'
         self.driver = webdriver.PhantomJS(executable_path=path)
+#        self.driver = webdriver.Firefox()
```

Comment:
this would make steve cross :)
alternatively, i can allow the driver to be choosable from the command line somehow if you like…